README.md

```diff
@@ -34,6 +34,26 @@ The barcode column can take values following the format barcode[0-9][0-9] or NB[
 
 Note: Basecalling and demultiplexing may also be performed if not already done on GridION/MinIT.
 
+## Basic usage using Dockerimage
+
+The Dockerimage is made to be used with Docker-out-of-Docker (DooD), where we mount `/var/run/docker.sock` into our container and our container will start new containers that are "siblings" to the one we started it from, and thus runs them in the context of the host. Due to this we have to use absolute paths when mounting volumes and setting `--input_dir` and `--sample_names`. 
+
+Assuming you have the sample_names.csv file in your `<run_name>` folder:
+
+``` shell
+docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /absolute/path/to/<run_name>:/absolute/path/to/<run_name>
+cov:test python susCovONT.py --yes --user_id 0 --input_dir /absolute/path/to/<run_name> --sample_names /absolute/path/to/<run_name>/sample_names.csv
```

Comment:
Is `cov:test` the docker image? Asking out of curiousity as I am not so familiar with docker.

Reply:
Very well spotted!

This is actually not the image we want to use, so I will update it to the proper one :)

Thank you!